src/pkg/packager/compose.go

```diff
@@ -145,11 +145,14 @@ func (p *Packager) fixComposedFilepaths(parent, child types.ZarfComponent) types
 		child.Files[fileIdx].Source = p.getComposedFilePath(file.Source, parent.Import.Path)
 	}
 
-	// Prefix non-url composed component chart values files.
+	// Prefix non-url composed component chart values files and localPath.
 	for chartIdx, chart := range child.Charts {
 		for valuesIdx, valuesFile := range chart.ValuesFiles {
 			child.Charts[chartIdx].ValuesFiles[valuesIdx] = p.getComposedFilePath(valuesFile, parent.Import.Path)
 		}
+		if child.Charts[chartIdx].LocalPath != "" {
+			child.Charts[chartIdx].LocalPath = p.getComposedFilePath(child.Charts[chartIdx].LocalPath, parent.Import.Path)
```

Comment:
```suggestion
			// Check if the localPath is relative to the parent Zarf package
            if _, err := os.Stat(child.Charts[chartIdx].LocalPath); os.IsNotExist(err) {
                // Since the chart localPath is not relative to the parent Zarf package, get the relative path from the composed child
                child.Charts[chartIdx].LocalPath = p.getComposedFilePath(child.Charts[chartIdx].LocalPath, parent.Import.path)
            }
```